fe/fe-core/src/main/java/com/starrocks/consistency/ConsistencyChecker.java

```diff
@@ -525,7 +525,11 @@ protected List<Long> chooseTablets() {
                                 // sort tablets
                                 Queue<MetaObject> tabletQueue =
                                         new PriorityQueue<>(Math.max(index.getTablets().size(), 1), COMPARATOR);
-                                tabletQueue.addAll(index.getTablets());
+                                long cooldownedTimeMs = startTime - Config.consistency_check_cooldown_time_second * 1000;
```

Comment:
The startTime here is confusing, because there is another param named startTime in the ConsistencyChecker class parameters. Can you change the tmp startTime to startCheckTime or something else?